lib/wings/valkyrie/query_service.rb

```diff
@@ -24,6 +24,18 @@ def find_by(id:)
         raise ::Valkyrie::Persistence::ObjectNotFoundError
       end
 
+      # Find all work/collection records, and map to Valkyrie Resources
+      # @return [Array<Valkyrie::Resource>]
+      def find_all
+        begin
```

Comment:
is the `begin` block needed?

Reply:
This was needed to get it to pass rubocop. Perhaps there is a better way to pacify rubocop?